ligo/stablecoin/actions.ligo

```diff
@@ -0,0 +1,288 @@
+(* SPDX-FileCopyrightText: 2020 tqtezos
+ * SPDX-License-Identifier: MIT *)
+
+(*
+ * Root entrypoints for stablecoin smart-contract
+ *)
+
+#include "operator.ligo"
+
+(* ------------------------------------------------------------- *)
+
+(*
+ * Helpers and validation
+ *)
+
+(*
+ * This function fails if provided token_id is not equal to
+ * default one, restricting all operations to be monolitic
```

Comment:
What do you mean by `monolithic` in this context?

Reply:
That are allowed for `default_token_id`